### ISSUE TITLE
Restructurize the code

### ISSUE THREAD

**Issue by Ickerday** [Nick-Gottschlich/Social-Amnesia#11]
Making myself acquainted with the code I found some articles with Tkinter best practices. Those along with some refactoring should reduce redundancy and make the code more easily extendable, say when the work would begin on a Facebook service. 
It's also a good occasion to learn Tkinter better.

Some proposed outlines:
- Encapsulation + docstrings instead of long methods with lots of comments (readability + sphinx support in the future?)
- more classes, 
- camelCase to pascal_case (and other standard code style stuff)
- removal of wildcard imports
- Type hints? (I'd love to finally see those in action, but those should probably come after the refactoring itself)

Anyone with more Python and Tkinter experience is more than welcome to help!

Articles:
https://www.begueradj.com/tkinter-best-practices.html
https://sukhbinder.wordpress.com/2014/12/25/an-example-of-model-view-controller-design-pattern-with-tkinter-python/
https://python-textbok.readthedocs.io/en/latest/Introduction_to_GUI_Programming.html
https://docs.python.org/3.7/library/tkinter.html#a-simple-hello-world-program

**Comment by Nick-Gottschlich** [comment]
I agree with this, one of the next things I was planning on working on was a big refactor. Looks like you might've beat me to it though 🙂. 

I'm primarily a Javascript developer, so I don't have a lot of experience with Python best practices (in fact, part of the purpose of building this was to learn Python better).

- Encapsulation + docstrings instead of long methods with lots of comments (readability + sphinx support in the future?) - Agreed
- more classes - agreed, although I don't know how classes work in Python at all, would be a learning experience for me
- camelCase to pascal_case (and other standard code style stuff) - Sure, we should probably be going with https://www.python.org/dev/peps/pep-0008/ (since as I understand it it's *the* python styling guide).
- removal of wildcard imports - agreed, I don't know how to do this though
- Type hints - I have a very limited understanding of this, but it seems to be a sort of static typing for Python? Seems interesting.

**Comment by Ickerday** [comment]
No worries, I just finished an apprenticeship where I worked in C# exclusively and I decided not to stay with C#. 

Most of the work concerning code style can be done with [PyCharm](https://www.jetbrains.com/pycharm/download) which is what I use when writing Python. I noticed you're using VSCode, it also has [support for linters](https://code.visualstudio.com/docs/python/linting). 

[Type hints](https://docs.python.org/3/library/typing.html) aren't used by the interpreter at all (yet) and won't ever be required for the program to work, but definitely help with autocomplete and make the dev experience better.
